python/nav/django/fields.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2018 UNINETT AS
```

Comment:
We already have a `nav.models.fields` for this purpose, is there any reason why we need another module?

Reply:
Model field vs. form field. We could move the actual field to `nav.django.forms` I guess.

Reply:
Moved in https://github.com/UNINETT/nav/pull/1684/commits/9285300d53af5cb016bc71f0dbe70706b0feb17b .